forge/test/operators/pytorch/reduce/errors/test_reduce_ids_tilize_error.txt

```diff
@@ -0,0 +1,76 @@
+E       RuntimeError: TT_FATAL @ /home/kmilanovic/src/ttforge/tt-forge-fe/third_party/tt-mlir/third_party/tt-metal/src/tt-metal/ttnn/cpp/ttnn/operations/data_movement/tilize_with_val_padding/device/tilize_with_val_padding_op.cpp:19: input_tensor_a.get_dtype() == DataType::BFLOAT16 or input_tensor_a.get_dtype() == DataType::UINT32
```

Comment:
Note: This error will be fixed in the next uplift of metal (fix was merged in metal yesterday).

Reply:
@dgolubovicTT  This error still occurs after i did an incremental build and even after complete cleanup and rebuild.